sdk/cloudmachine/Azure.CloudMachine.OpenAI/src/OpenAIModelFeature.cs

```diff
@@ -118,7 +118,7 @@ protected override ProvisionableResource EmitResources(CloudMachineInfrastructur
             },
             Sku = new CognitiveServicesSku
             {
-                Capacity = 120,
+                Capacity = 10,
```

Comment:
you'll need to update these test biceps here [https://grep.app/search?q=AllowBlobPublicAccess&filter[repo][0]=Azure/azure-sdk-for-net&filter[path][0]=sdk/cloudmachine/Azure.Provisioning.CloudMachine/tests/Data/](https://grep.app/search?q=AllowBlobPublicAccess&filter%5Brepo%5D%5B0%5D=Azure/azure-sdk-for-net&filter%5Bpath%5D%5B0%5D=sdk/cloudmachine/Azure.Provisioning.CloudMachine/tests/Data/)